week-1/interpret/percentage-change.js

```diff
@@ -9,12 +9,28 @@ const percentageChange = (priceDifference / carPrice) * 100;
 
 console.log(`The percentage change is ${percentageChange}`);
 
+
 // Read the code and then answer the questions below
 
 // a) How many function calls are there in this file? Write down all the lines where a function call is made
 
+// 2 functions : 
+// Number(carPrice.replaceAll(",", ""))
+// Number(priceAfterOneYear.replaceAll(",", ""))
+
 // b) Identify all the lines that are variable reassignment statements
 
+// 3 variable reassignment statements : carPrice = Number(carPrice.replaceAll(",", ""));
+// priceAfterOneYear = Number(priceAfterOneYear.replaceAll(",", ""));
+// percentageChange = (priceDifference / carPrice) * 100;
```

Comment:
The question asked for varible reassignment statements. One of these is in fact an intial reassignment. Which one is it?

Reply:
is it this one "carPrice = Number(carPrice.replaceAll(",", ""))"?